acceptance-tests/src/test/java/gov/di_ipv_drivingpermit/pages/DrivingLicencePageObject.java

```diff
@@ -1008,4 +1017,17 @@ public void assertDVLAConsentErrorOnCheckbox(String expectedText) {
         Assert.assertEquals(
                 expectedText, DVLAConsentCheckboxError.getText().trim().replace("\n", ""));
     }
+
+    public void assertConsentSection(String consentSection) {
+
+        Assert.assertEquals(consentSection, DVLAConsentSection.getText());
+    }
+
+    public void assertOneLoginPrivacyLink(String oneLoginPrivacyLink) {
+        Assert.assertEquals(oneLoginPrivacyLink, oneLoginLink.getText());
```

Comment:
Should we not be asserting the link location as well as the text. Can we also make a not on lime 562 that this is a prime test for frontend once we have it ready